priv/repo/migrations/20160520141230_add_token_to_member.exs

```diff
@@ -0,0 +1,11 @@
+defmodule Community.Repo.Migrations.AddTokenToMember do
+  use Ecto.Migration
+
+  def change do
+    execute "TRUNCATE TABLE members;"
```

Comment:
1. We have no members. This is the easy solution.
2. Tokens can't be null. This is the easy solution to prevent that from being a problem.